contracts/contracts/harvest/Harvester.sol

```diff
@@ -0,0 +1,337 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity ^0.8.0;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { SafeMath } from "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/math/Math.sol";
+
+import { StableMath } from "../utils/StableMath.sol";
+import { Initializable } from "../utils/Initializable.sol";
+import { Governable } from "../governance/Governable.sol";
+import { IVault } from "../interfaces/IVault.sol";
+import { IOracle } from "../interfaces/IOracle.sol";
+import { IStrategy } from "../interfaces/IStrategy.sol";
+import { IUniswapV2Router } from "../interfaces/uniswap/IUniswapV2Router02.sol";
+import "../utils/Helpers.sol";
+
+contract Harvester is Initializable, Governable {
+    using SafeERC20 for IERC20;
+    using SafeMath for uint256;
+    using StableMath for uint256;
+
+    // Tokens that should be swapped for stablecoins
+    address[] public swapTokens;
+
+    // Address of Uniswap
+    address public uniswapAddr = address(0);
+
+    // Address of Vault
+    address public vaultAddress = address(0);
+
+    // Reserved for future expansion
+    int256[100] private _reserved;
```

Comment:
We only need this reserved if something else is going to inherit from it.

Reply:
Good point removing it